WikiTemplate/iOS/wikitemplate-ios.md

```diff
@@ -90,46 +108,63 @@
+- [ ] Verify history sync is disabled by default. Enabling it starts sync'ing history items as well (limited to 200 entries)
 - [ ] Verify sync works on an upgraded profile and new bookmarks added post-upgrade sync across devices on the chain
 - [ ] Verify you can create a standalone Sync chain with one device
+- [ ] Verify starting sync chain using Other Devices segmented tab. Ensured enabling `open tabs` will display tabs from other devices
+- [ ] Verify sharing pages with `Send To Your Devices`
+- [ ] Verify `Sync Internals` page data is populated when connected to sync chain
+- [ ] Verify when using `Delete Sync Account` will remove all devices connected to sync chain
 
 ## Playlist & Autoplay
 
 - [ ] Verify Playlist is enabled by default on a clean install
 - [ ] Verify when a site with audio/video is detected, `Playlist` button is shown in URL bar and a notification badge on the menu
 - [ ] Verify clicking the `Playlist` button adds audio/video to playlist and initiates offline play
-- [ ] Verify able to add audio/video to playlist via share menu
-- [ ] Verify able to add audio/video to playlist via long press context menu
-- [ ] Verify Autoplay is disabled by default
+- [ ] Verify adding audio/video to playlist via share menu
+- [ ] Verify adding audio/video to playlist via long press context menu
+- [ ] Verify Autoplay is enabled by default
 - [ ] Verify with Autoplay enabled, able to play audio/video automatically
 - [ ] Verify with background audio enabled, able to play audio/video with device locked or app in background
+- [ ] Verify Playlist media works with CarPlay on iOS devices
+- [ ] Verify playback gestures are working in Playlist
+- [ ] Verify adding shared folder to playlist using `https://playlist.brave.com/bsa-sample`. Verify all actions on shared folder are working
 
 ## Brave News
 
 - [ ] Verify Brave News opt-in is shown on a clean install
 - [ ] Verify once opted-in Brave News starts showing articles
-- [ ] Verify promoted content is shown on Brave News
 - [ ] Verify inline-content ads are shown on Brave News
-- [ ] Verify able to add custom RSS feed to the list of sources
-- [ ] Verify able to disable content from a particular publisher by long-press on the card
+- [ ] Verify able to add custom RSS feed to the list of sources. Ensure this is working when entering custom URL manually and through the share menu when at a blog page
+- [ ] Verify able to hide content from sources within the `Popular Sources` and `Suggested` categories by long-press on the card
+- [ ] Verify adding Top News Brave widget to home screen will display news articles. Ensure tapping on news articles will load in brave.
+
+## VPN
+
+- [ ] Verify purchasing VPN subscription will install VPN profile and establish connection
+- [ ] Verify tunneling works by navigating to `https://ipinfo.io`
+- [ ] Verify changing server region will update VPN connection
```

Comment:
Suggestion

add `works as expected` to the end of sentence

Reply:
See last comment.